doc/developer-guides/trusty.rst

```diff
@@ -0,0 +1,186 @@
+.. _trusty_tee:
+
+Trusty TEE on ACRN
+##################
+
+Introduction
+************
+
+`Trusty`_ is a set of software components supporting a Trusted Execution Environment (TEE).
+Trusty consists of:
+
+1. An operating system (the Trusty OS) that runs on a processor intended to provide a TEE
+#. Drivers for the Android kernel (Linux) to facilitate communication with applications
+   running under the Trusty OS
+#. A set of libraries for Android/Linux systems software to facilitate communication with
+   trusted applications executed within the Trusty OS using the kernel drivers
+
+LK (`Little Kernel`_) is a tiny operating system suited for small embedded devices, bootloaders,
+and other environments where OS primitives like threads, mutexes, and timers are needed, but
+there's a desire to keep things small and lightweight. LK has been chosen as the Trusty OS kernel.
+
+Trusty Architecture
+*******************
+
+Trusty Architectural diagram::
+
+                      +---------------------------+
+                      |VMn                        |
+                      |        ......             |
+    +------------+  +---------------------------+ |
+    |VM0         |  |VM1                        | |
+    |            |  | +--------+     +--------+ | |
+    |            |  | |        |     |        | | |
+    |    SOS     |  | | Normal |     | Secure | | |
+    |            |  | | World  |     | World  | |-+
+    |            |  | |        |     |        | |
+    |            |  | +--------+     +--------+ |
+    +------------+  +---------------------------+
+    +-------------------------------------------+
+    |             ACRN Hypervisor               |
+    +-------------------------------------------+
+    +-------------------------------------------+
+    |                   HW                      |
+    +-------------------------------------------+
+
+.. note::
+   Trusty OS is running in Secure World in the architecture above.
```

Comment:
change the sentence ending to:

    ... in the architecture drawing above.

Reply:
OK